framework/wazuh/rbac/orm.py

```diff
@@ -864,6 +864,8 @@ def update_user(self, user_id: int, password: str = None, name: str = None) -> b
             Password provided by user. It will be stored hashed.
         name : str
             New username.
+        hashed : bool
+            Hash the password before saving it to the database if `False`. Save it raw otherwise.
```

Comment:
The description and usage of this variable are confusing. The description says to hash the password before saving it if false, or save it raw otherwise, but in this case, if hashed is true it means the password is already hashed, and not raw, and that's why it is saved as is.

Also, to be consistent, the name of the variable should be the same as the one used in the add_user method, where the description is also inconsistent.